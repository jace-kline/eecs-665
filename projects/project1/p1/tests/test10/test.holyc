"Testing whether the unterminated error works" 123 "
Hello I am Jace "This is another test \x
true false boolptr