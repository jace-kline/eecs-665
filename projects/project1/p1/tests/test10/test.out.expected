STRINGLIT:"Testing whether the unterminated error works" [1,1]
INTLIT:123 [1,48]
ID:Hello [2,1]
ID:I [2,7]
ID:am [2,9]
ID:Jace [2,12]
TRUE [3,1]
FALSE [3,6]
BOOLPTR [3,12]
EOF [3,19]
